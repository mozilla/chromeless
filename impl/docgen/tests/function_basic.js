/**
 * A module with a basic function definition in it
 */

/**
 * @function toFilename
 * a function which converts URLs to filenames
 */
