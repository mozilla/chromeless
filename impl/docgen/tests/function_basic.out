{
  "desc": "A module with a basic function definition in it", 
  "filename": "tests/function_basic.js", 
  "functions": [
    {
      "desc": "a function which converts URLs to filenames", 
      "name": "toFilename"
    }
  ], 
  "module": "function_basic"
}
